http/handler.go

```diff
@@ -314,17 +314,16 @@ func handleAuditNonLogical(core *vault.Core, h http.Handler) http.Handler {
 		cw := newCopyResponseWriter(w)
 		h.ServeHTTP(cw, r)
 		data := make(map[string]interface{})
-		err = jsonutil.DecodeJSON(cw.body.Bytes(), &data)
-		if err != nil {
-			// best effort, ignore
-		}
+
+		// Refactoring this code, since the returned error was being ignored.
```

Comment:
This makes sense.
I'm curious about why we do this decoding in the first place though. Should we just delete it?

Reply:
Ah wait nvm. Looks like we use data in the next line.